### PR TITLE
Readme overhaul 2020-11

### PR DESCRIPTION
**Pull request by alexcg1**
- contents moved to menu bar
- simplify sectioning
- tweak wording

### CODE REVIEW

**Comment by hanxiao** [comment]
can you resolve the conflicts? also i see you remove the TOC, better keep it.

**Comment by alexcg1** [comment]
Re-added contents @hanxiao 

Reason I removed in the first place was bc already included many links above

File: README.md [hunk 1]
```diff
@@ -26,34 +25,26 @@ An easier way to build neural search in the cloud
...
+</sub>
+</p>
 
-Jina is an AI-powered search framework, empowering developers to create **cross-/multi-modal search systems** (e.g. text, images, video, audio) in the cloud. Jina is supported long-term by a [full-time, venture-backed team](https://jina.ai).
+<p align="center">
+Jina is a deep-learning-powered search framework for building <strong>cross-/multi-modal search systems</strong> (e.g. text, images, video, audio) in the cloud. 
+</p>
 
-⏱️ **Time Saver** - Bootstrap an AI-powered system in just a few minutes.
+⏱️  **Time Saver** - Bootstrap an AI-powered system in just a few minutes.
```
Comment (hanxiao) [inline README.md:35]:
```suggestion
⏱️ **Time Saver** - Bootstrap an AI-powered system in just a few minutes.
```

File: README.md [hunk 2]
```diff
@@ -46,14 +42,13 @@ Jina is a deep-learning-powered search framework for building **cross-/multi-mod
...
-✔️ **Here to Stay** - Jina is supported long-term by a [full-time, venture-backed team](https://jina.ai).
+❤️  **Made with Love** - Quality first, with no code compromises, delivered by a [full-time, venture-backed team](https://jina.ai).
 
 ---
 
-
 ## Quick Start
 
-On Linux/MacOS with Python 3.7/3.8:
+On Linux/macOS with Python 3.7/3.8:
```
Comment (theUnkownName) [inline README.md:70]: Just curious why was this changed not to use capital letter for Mac?

Comment (alexcg1) [reply README.md:70]: That's because MacOS is now called macOS by Apple

Comment (alexcg1) [reply README.md:70]: https://www.apple.com/macos/catalina/

Comment (theUnkownName) [reply README.md:70]: Ohhh didn't know, cool!

File: README.md [hunk 3]
```diff
@@ -225,16 +225,6 @@ jina hello-world --help
...
-### Create Your First Jina Project
-
-```bash
-pip install jina[devel]
-jina hub new --type app
-```
-
-You can easily create a Jina project from templates with one terminal command. This creates a Python entrypoint, YAML configs and a Dockerfile. You can start from there.
-
-
```
Comment (hanxiao) [inline README.md:237]: keep it for now, also leave from hello-world to this section to me.

File: README.md [hunk 4]
```diff
@@ -74,9 +65,9 @@ Jina is an AI-powered search framework, empowering developers to create **cross-
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-## Install
+## Quick Start
```
Comment (hanxiao) [inline README.md:68]:
```suggestion
## Install
```

**Comment by alexcg1** [comment]
Getting some errors from `commitlint`
Screenshot from 2020-11-11 15-37-25

I've run `git log` but I can't see the commit in there

**Comment by alexcg1** [comment]
I think bc I clicked a button to commit the changes @hanxiao requested
